run.go

```diff
@@ -734,3 +744,18 @@ func propagateContainerdLabelsToOCIAnnotations() oci.SpecOpts {
 		return oci.WithAnnotations(c.Labels)(ctx, oc, c, s)
 	}
 }
+
+func writeCIDFile(path, id string) error {
+	if _, err := os.Stat(path); err == nil {
+		return errors.Errorf("container ID file found, make sure the other container isn't running or delete %s", path)
+	}
+	f, err := os.Create(path)
```

Comment:
I think you should do smthing like this to handles err différent from `isNotExist`

```
if _, err := os.Stat("/path/to/whatever"); err == nil {
  // throw erro
} else if os.IsNotExist(err) {
  // your logic here

} else {
  // handle other erro
}
```

Reply:
I added returning other errors, please check if this way makes sense. Also I used `errors.Is` according to https://golang.org/pkg/os/#IsExist